### PR TITLE
Update tutorial.md

### PR DESCRIPTION
**Pull request by subinamehta**
Made few updates to include tool icons while running GTN inside Galaxy interface

### CODE REVIEW

File: topics/proteomics/tutorials/proteogenomics-dbsearch/tutorial.md [hunk 1]
```diff
@@ -222,7 +222,7 @@ The mzidentml output from the Peptide shaker is converted into an sqlite databas
 >
 > This tool extracts mzidentml and its associated proteomics datasets into a sqlite db
 >
-> 1. **mz to sqlite** {% icon tool %} with the following parameters:
+> 1. {% tool [**mz to sqlite**](toolshed.g2.bx.psu.edu/repos/galaxyp/mz_to_sqlite/mz_to_sqlite/2.0.4+galaxy1) %} with the following parameters:
```
Comment (hexylena) [inline topics/proteomics/tutorials/proteogenomics-dbsearch/tutorial.md:225]:
```suggestion
> 1. {% tool [mz to sqlite](toolshed.g2.bx.psu.edu/repos/galaxyp/mz_to_sqlite/mz_to_sqlite/2.0.4+galaxy1) %} with the following parameters:
```

File: topics/proteomics/tutorials/proteogenomics-novel-peptide-analysis/tutorial.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ The first step in this tutorial is to perfrom BLAST-P analysis using the NCBI-NR
 
 > ### {% icon hands_on %} Hands-on: NCBI BLAST+ blastp
 >
-> 1. **NCBI BLAST+ blastp** {% icon tool %} with the following parameters:
+> 1. {% tool [**NCBI BLAST+ blastp**](toolshed.g2.bx.psu.edu/repos/devteam/ncbi_blast_plus/ncbi_blastp_wrapper/0.3.3) %} with the following parameters:
```
Comment (hexylena) [inline topics/proteomics/tutorials/proteogenomics-novel-peptide-analysis/tutorial.md:83]:
```suggestion
> 1. {% tool [NCBI BLAST+ blastp](toolshed.g2.bx.psu.edu/repos/devteam/ncbi_blast_plus/ncbi_blastp_wrapper/0.3.3) %} with the following parameters:
```

File: topics/proteomics/tutorials/proteogenomics-novel-peptide-analysis/tutorial.md [hunk 3]
```diff
@@ -110,7 +110,7 @@ Once Blast-P search is performed, it provides a tabular output containing “**N
 
 > ### {% icon hands_on %} Hands-on: Query Tabular
 >
-> 1. **Query Tabular** {% icon tool %} with the following parameters:
+> 1. {% tool [**Query Tabular**](toolshed.g2.bx.psu.edu/repos/iuc/query_tabular/query_tabular/3.0.0) %} with the following parameters:
```
Comment (hexylena) [inline topics/proteomics/tutorials/proteogenomics-novel-peptide-analysis/tutorial.md:113]:
```suggestion
> 1. {% tool [Query Tabular](toolshed.g2.bx.psu.edu/repos/iuc/query_tabular/query_tabular/3.0.0) %} with the following parameters:
```

File: topics/proteomics/tutorials/proteogenomics-novel-peptide-analysis/tutorial.md [hunk 4]
```diff
@@ -166,7 +166,7 @@ Once this step is completed, a tabular output containing novel proteoforms are d
 
 > ### {% icon hands_on %} Hands-on: Query Tabular
 >
-> 1. **Query Tabular** {% icon tool %}: with the following parameters:
+> 1. {% tool [**Query Tabular**](toolshed.g2.bx.psu.edu/repos/iuc/query_tabular/query_tabular/3.0.0) %}
```
Comment (hexylena) [inline topics/proteomics/tutorials/proteogenomics-novel-peptide-analysis/tutorial.md:169]:
```suggestion
> 1. {% tool [Query Tabular](toolshed.g2.bx.psu.edu/repos/iuc/query_tabular/query_tabular/3.0.0) %}
```

File: topics/proteomics/tutorials/proteogenomics-novel-peptide-analysis/tutorial.md [hunk 5]
```diff
@@ -293,7 +293,7 @@ peptides provided as input. This outputs bed file for peptides.
 
 > ### {% icon hands_on %} Hands-on: Peptide genomic Coordinate
 >
-> 1. Run **Peptide genomic Coordinate** {% icon tool %} with the following parameters:
+> 1. Run {% tool [**Peptide genomic Coordinate**] (toolshed.g2.bx.psu.edu/repos/galaxyp/peptide_genomic_coordinate/peptide_genomic_coordinate/0.1.1) %} with the following parameters:
```
Comment (hexylena) [inline topics/proteomics/tutorials/proteogenomics-novel-peptide-analysis/tutorial.md:296]:
```suggestion
> 1. Run {% tool [Peptide genomic Coordinate] (toolshed.g2.bx.psu.edu/repos/galaxyp/peptide_genomic_coordinate/peptide_genomic_coordinate/0.1.1) %} with the following parameters:
```

File: topics/proteomics/tutorials/proteogenomics-novel-peptide-analysis/tutorial.md [hunk 6]
```diff
@@ -311,7 +311,7 @@ Given chromosomal locations of peptides in a BED file, PepPointer classifies the
 
 > ### {% icon hands_on %} Hands-on: Peppointer
 >
-> 1. **Peppointer** {% icon tool %} with the following parameters:
+> 1. {% tool [**Peppointer**](toolshed.g2.bx.psu.edu/repos/galaxyp/pep_pointer/pep_pointer/0.1.3) %} with the following parameters:
```
Comment (hexylena) [inline topics/proteomics/tutorials/proteogenomics-novel-peptide-analysis/tutorial.md:314]:
```suggestion
> 1. {% tool [Peppointer](toolshed.g2.bx.psu.edu/repos/galaxyp/pep_pointer/pep_pointer/0.1.3) %} with the following parameters:
```

File: topics/proteomics/tutorials/proteogenomics-novel-peptide-analysis/tutorial.md [hunk 7]
```diff
@@ -331,7 +331,7 @@ The final tool for this workflow generates a tabular output that summarizes the
 
 > ### {% icon hands_on %} Hands-on: Query Tabular
 >
->  1. **Query Tabular** {% icon tool %} with the following parameters:
+>  1. {% tool [**Query Tabular**](toolshed.g2.bx.psu.edu/repos/iuc/query_tabular/query_tabular/3.0.0) %}
```
Comment (hexylena) [inline topics/proteomics/tutorials/proteogenomics-novel-peptide-analysis/tutorial.md:334]:
```suggestion
>  1. {% tool [Query Tabular](toolshed.g2.bx.psu.edu/repos/iuc/query_tabular/query_tabular/3.0.0) %}
```

**Review by hexylena — DISMISSED** [review]
This looks good @subinamehta! I've made a couple small fixes in the files changed tab if you want to apply them. The new syntax doesn't require making tool names bold anymore! that's now done automatically

**Comment by hexylena** [comment]
It looks like there was an extra space between the ] and (, but if that doesn't fix it I'll look into it!

**Review by hexylena — APPROVED** [review]

**Review by shiltemann — APPROVED** [review]

**Comment by shiltemann** [comment]
thanks @subinamehta!

**Pull request merged**